include/cantera/kinetics/StoichManager.h

```diff
@@ -635,11 +635,11 @@ class StoichManagerN
         m_values.resize(nCoeffs, 0.);
 
         // Set up index pairs for derivatives
-        std::map<std::pair<int, int>, size_t> indices;
-        size_t n = 0;
+        std::map<std::pair<size_t, size_t>, int> indices;
+        int n = 0;
         for (int i = 0; i < tmp.outerSize(); i++) {
             for (Eigen::SparseMatrix<double>::InnerIterator it(tmp, i); it; ++it) {
-                indices[{it.row(), it.col()}] = n++;
+                indices[{(size_t)it.row(), (size_t)it.col()}] = n++;
```

Comment:
This is quite a bit more verbose, but I've seen it strongly recommended to avoid the C-style cast in C++:
```suggestion
                indices[{static_cast<size_t>(it.row()), static_cast<size_t>(it.col())}] = n++;
```